test/integration/api.js

```diff
@@ -262,7 +262,11 @@ describe('api v.1', () => {
                 user_id: user.get('id'),
                 text: `This is a Post #${i}`
               });
-              return post.save({fully_published_at: (new Date(Date.now() - 50000 + i*1000)).toJSON()}, {method: 'insert'});
+
+              // even posts with fully_published_at, odd posts without fully_published_at, only with created_at
+              let attr = i % 2 ? {fully_published_at: (new Date(Date.now() - 50000 + i*1000)).toJSON()} : {
+                created_at: i % 2 == 0 ? (new Date(Date.now() - 50000 + i*1000)).toJSON() : null};
```

Comment:
this is 100% unreadable. please:

1. extract timestamp into constant
2. reformat and get rid of nested ternary. that's evil